securedrop/tests/test_journalist.py

```diff
@@ -1585,6 +1586,39 @@ def test_render_locales(config, journalist_app, test_journo, test_source):
     assert url_end + '?l=en_US' in text, text
 
 
+def test_render_xenial_positive(config, journalist_app, test_journo):
+    yesterday = datetime.datetime.now() - datetime.timedelta(days=1)
+    journalist_app.config.update(
+        XENIAL_WARNING_DATE=yesterday,
+        XENIAL_VER='16.04'
+    )
+
+    with journalist_app.test_client() as app:
+        _login_user(app, test_journo['username'], test_journo['password'],
+                    test_journo['otp_secret'])
+
+        resp = app.get(url_for('main.index'))
+
+    text = resp.data.decode('utf-8')
+    assert "critical-skull" in text, text
+
+
+def test_render_xenial_negative(config, journalist_app, test_journo):
+    yesterday = datetime.datetime.now() - datetime.timedelta(days=1)
+    journalist_app.config.update(
+        XENIAL_WARNING_DATE=yesterday,
+        XENIAL_VER='14.04'
+    )
+
+    with journalist_app.test_client() as app:
+        _login_user(app, test_journo['username'], test_journo['password'],
+                    test_journo['otp_secret'])
+        resp = app.get('/')
+
+    text = resp.data.decode('utf-8')
+    assert "critical-skull" not in text, text
+
+
```

Comment:
What would you think about adding another negative test scenario (the one where the warning date is in the future), eg:

```
diff --git a/securedrop/tests/test_journalist.py b/securedrop/tests/test_journalist.py
index fd548e563..d189ba656 100644
--- a/securedrop/tests/test_journalist.py
+++ b/securedrop/tests/test_journalist.py
@@ -1603,7 +1603,7 @@ def test_render_xenial_positive(config, journalist_app, test_journo):
     assert "critical-skull" in text, text
 
 
-def test_render_xenial_negative(config, journalist_app, test_journo):
+def test_render_xenial_negative_version(config, journalist_app, test_journo):
     yesterday = datetime.datetime.now() - datetime.timedelta(days=1)
     journalist_app.config.update(
         XENIAL_WARNING_DATE=yesterday,
@@ -1618,6 +1618,20 @@ def test_render_xenial_negative(config, journalist_app, test_journo):
     text = resp.data.decode('utf-8')
     assert "critical-skull" not in text, text
 
+def test_render_xenial_negative_date(config, journalist_app, test_journo):
+    tomorrow = datetime.datetime.now() + datetime.timedelta(days=1)
+    journalist_app.config.update(
+        XENIAL_WARNING_DATE=tomorrow,
+        XENIAL_VER='14.04'
+    )
+
+    with journalist_app.test_client() as app:
+        _login_user(app, test_journo['username'], test_journo['password'],
+                    test_journo['otp_secret'])
+        resp = app.get('/')
+
+    text = resp.data.decode('utf-8')
+    assert "critical-skull" not in text, text
```

Reply:
will do!